desktop-widgets/tab-widgets/TabDiveInformation.ui

```diff
@@ -445,23 +445,27 @@
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="diveInfoWaterDensityLayout">
-
           <item>
-        <widget class="QLabel" name="waterTypeText">
+           <widget class="QLabel" name="waterTypeText">
+           </widget>
+          </item>
+          <item>
+           <widget class="QComboBox" name="waterTypeCombo"/>
+          </item>
+          <item>
+           <widget class="QLabel" name="salinityOverWrittenIcon">
          <property name="text">
-          <string>EN13319</string>
+          <string>!</string>
```

Comment:
why is the text now `!` ?

Reply:
I hope this is solved. This is not a bug, it is meant to be like that.